models/analytics.go

```diff
@@ -21,7 +32,21 @@ type Analytics struct {
 	SignedEmbeddingURL string
 }
 
-type AnalyticsCustomClaims struct {
-	Resource map[string]interface{}
-	Params   map[string]interface{}
+type AnalyticsCustomClaims interface {
+	GetEmbeddingType() EmbeddingType
+	GetParams() map[string]interface{}
+}
+
+type GlobalDashboardAnalytics struct {
+	OrganizationId string
+}
+
+func (analytics GlobalDashboardAnalytics) GetEmbeddingType() EmbeddingType {
+	return GlobalDashboard
+}
+
+func (analytics GlobalDashboardAnalytics) GetParams() map[string]interface{} {
+	return map[string]interface{}{
+		"organization": []string{analytics.OrganizationId},
```

Comment:
previously it seemed to be org_id, was it a change on metabase ?

Reply:
This part depends on the way embeding is configured.
The last one I tested was using `organization` other `org_id`.

This kind of divergence is one of the reason I moved to a "interface" style directly inside `models.analytics` so each metabase embedding can be easilly customised looking at a single file.

Basically, I do the following :

1. Ask for the Metabase embedding link
2. Open the embedding window from the interface
3. Look at the code section
4. Copy/Paste the information to implement the new embedding as the Go interface described in models (e.g. embeding type, params form...)